CMakeLists.txt

```diff
@@ -18,6 +18,8 @@ if(CONFIG_HAS_NRFX)
   zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF52811      NRF52811_XXAA)
   zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF52832      NRF52832_XXAA)
   zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF52840      NRF52840_XXAA)
+  zephyr_compile_definitions_ifdef(CONFIG_SOC_NRF5340_CPU0  NRF5340_XXAA_APPLICATION)
```

Comment:
Please change to CPUAPP and CPUNET, respectively. This aligns with using "app" and "net" in the nRF53 SoC and Board definitions.